src/components/dar_dataset_table/DarDatasetTable.js

```diff
@@ -182,14 +181,12 @@ export const DarDatasetTable = (props) => {
         setBuckets([]);
         return;
       }
-      const processedBuckets = await flow([
-        generatePreProcessedBucketData,
-        processDataUseBuckets,
-      ])({ dars, datasets });
-      const filteredDataUseBuckets = processedBuckets.filter(
+      const binnedBuckets = await binCollectionToBuckets(collection);
+      const filteredBinnedBuckets = binnedBuckets.filter(
```

Comment:
There's a bug directly above this block. Lines 165-179 effectively remove dars/datasets from the buckets if the user doesn't have any votes in them. I need to add the same filtering logic in the voting pages to this page as well.